manifest.json

```
@@ -1417,6 +1417,13 @@
       "type": "library",
       "remote": "https://github.com/adamharrison/lite-xl-www.git:aecc866c33b0a954fbd41a5de2842bb97f4ac45e",
       "mod_version": "3"
+    },
+    {
+     "id": "code+",
+     "version": "1.0",
+     "description": "Offers improvements such as highlighted comments and autocomplete for brackets, quotes and more. *([demo](https://s12.gifyu.com/images/Sckre.gif))*",
+     "remote": "https://github.com/chqs-git/code-plus",
```

You must specify a commit hash like:
```suggestion
     "remote": "https://github.com/chqs-git/code-plus:b336999b707709df2ddcb865f664623c5beceabb",
```

---

Out of curiosity can I ask why?

---

Just understood, thank you.